Wire-iOS/Sources/UserInterface/Conversation Options/ConversationServicesOptionsViewModel.swift

```diff
@@ -63,7 +64,16 @@ final class ConversationServicesOptionsViewModel {
         self.configuration = configuration
         state.title = configuration.title
         updateRows()
-}
+
+        configuration.allowServicesChangedHandler = { [weak self] allowServices in
+            guard let `self` = self else { return }
+            if allowServices {
+                self.updateRows()
+            } else {
+                self.updateRows()
+            }
```

Comment:
You can replace this with the equivalent:

```suggestion
            self?.updateRows()
            }
```